test/testrunner.py

```diff
@@ -351,8 +351,9 @@ def main():
                 context = Context()
                 context.run("git clone https://github.com/HabanaAI/gaudi-test-suite.git")
                 context.run("tar -c -f gaudi-test-suite.tar.gz gaudi-test-suite")
-
-            pytest_cmd += ["--reruns=1", "--reruns-delay=10"]
+            else:
+                pytest_cmd += ["--reruns=1", "--reruns-delay=10"]
```

Comment:
Few Habana tests execute for longer that 2 hours. The entire process of creating instance, downloading things and running script takes more than 2 hours. At times, if there is a failure, it reruns. When it runs the second time, the CodeBuild job stops executing and fails to show the logs. In other words, after 4 to 5 hours, we do not see the failure logs. Preventing the retry would be better for Habana tests as the tests are huge.